src/main/scala/com/fulcrumgenomics/cmdline/FgBioMain.scala

```diff
@@ -54,7 +57,8 @@ object FgBioMain {
 class FgBioCommonArgs
 ( @arg(doc="Use asynchronous I/O where possible, e.g. for SAM and BAM files.") val asyncIo: Boolean = false,
   @arg(doc="Default GZIP compression level, BAM compression level.")           val compression: Int = 5,
-  @arg(doc="Directory to use for temporary files.")                            val tmpDir: DirPath  = Paths.get(System.getProperty("java.io.tmpdir"))
+  @arg(doc="Directory to use for temporary files.")                            val tmpDir: DirPath  = Paths.get(System.getProperty("java.io.tmpdir")),
+  @arg(doc="Minimum severity log-level to emit.")                              val logLevel: LogLevel = LogLevel.Info
```

Comment:
It'd be nice if the message was the same as `dagr` ;)

Reply:
I like this message better ;)  Maybe I'll PR the message in dagr.